src/pages/workspaces/workspace/Data.js

```diff
@@ -456,10 +459,10 @@ class EntitiesContent extends Component {
       workspace, workspace: { workspace: { namespace, name }, workspaceSubmissionStats: { runningSubmissionsCount } },
       entityKey, entityMetadata, loadMetadata, firstRender
     } = this.props
-    const { selectedEntities, deletingEntities, copyingEntities, refreshKey, igvFiles, showIgvSelector } = this.state
+    const { selectedEntities, deletingEntities, copyingEntities, refreshKey, igvData } = this.state
```

Comment:
```suggestion
    const { selectedEntities, deletingEntities, copyingEntities, refreshKey, igvData: { selectedFiles, refGenome, showIgvSelector } } = this.state
```

If you do this, you can simplify the references to this data below again.

Reply:
Awesome, thanks @zarsky-broad